src/devices/MettlerToledo/samples/README.md

```diff
@@ -0,0 +1,4 @@
+# TODO: This needs to be determined
+
+Help Wanted Please
```

Comment:
What kind of help do you need? :-) You can refer to the main readme as you placed the example and detailed usage there

Reply:
Oops, this is part of the template